tests/e2e/env/shared.sh

```diff
@@ -8,23 +8,29 @@ export WP_URL="localhost:8084"
 export SERVER_PATH="$E2E_ROOT/deps/wcp-server"
 export DEV_TOOLS_DIR="wcp-dev-tools"
 export DEV_TOOLS_PATH="$E2E_ROOT/deps/$DEV_TOOLS_DIR"
+export WP_CONTAINER="wcp_e2e_wordpress"
 
 step() {
 	echo
 	echo "===> $1"
 }
 
 redirect_output() {
-	if [ -z "$DEBUG" ]; then
-        "$@" > /dev/null
-    else
+	if [[ "$DEBUG" = true ]]; then
         "$@"
+    else
+        "$@" > /dev/null
     fi
 }
 
 # --user xfs forces the wordpress:cli container to use a user with the same ID as the main wordpress container. See:
 # https://hub.docker.com/_/wordpress#running-as-an-arbitrary-user
 cli()
 {
-	redirect_output docker run -i --rm --user xfs --volumes-from $WP_CONTAINER --network container:$WP_CONTAINER wordpress:cli "$@"
+	redirect_output docker run -i --rm --user xfs --volumes-from "$WP_CONTAINER" --network container:"$WP_CONTAINER" wordpress:cli "$@"
+}
+
+cli_debug()
+{
+	docker run -i --rm --user xfs --volumes-from "$WP_CONTAINER" --network container:"$WP_CONTAINER" wordpress:cli "$@"
```

Comment:
Could you add a comment explaining the bifurcation for `redirect_output`?

Reply:
Done 👍